src/gui/virtualstudio.cpp

```diff
@@ -933,6 +1000,189 @@ void VirtualStudio::setupAuthenticator()
     }
 }
 
+QString VirtualStudio::randomString(int stringLength)
+{
+    QString str        = "";
+    static bool seeded = false;
+    QString allow_symbols(
+        "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789");
+
+    if (!seeded) {
+        qsrand(QTime::currentTime().msec());
+        seeded = true;
+    }
+
+    for (int i = 0; i < stringLength; ++i) {
+        str.append(allow_symbols.at(qrand() % (allow_symbols.length())));
+    }
+
+    return str;
+}
+
+void VirtualStudio::registerJTAsDevice()
+{
+    /*
+        REGISTER JT APP AS A DEVICE ON VIRTUAL STUDIO
+
+        Defaults:
+        period - 128 - set by studio = buffer size
+        queueBuffer - 0 - set by studio = net queue
+        devicePort - 4464
+        reverb - 0 - off
+        limiter - false
+        compressor - false
+        quality - 2 - high
+        captureBoost - false
+        captureMute - false - unused right now
+        captureVolume - 100 - unused right now
+        playbackBoost - false
+        playbackMute - false - unused right now
+        playbackVolume - 100 - unused right now
+        monitorMute - false - unsure if we should enable
+        monitorVolume - 0 - unsure if we should enable
+        name - "JackTrip App"
+        alsaName - "jacktripapp"
+        overlay - "jacktrip_app"
+        mac - UUID tied to app session
+        version - app version - will need to update in heartbeat
+        apiPrefix - random 7 character string tied to app session
+        apiSecret - random 22 character string tied to app session
+    */
+
+    QJsonObject json = {
+        {QLatin1String("period"), 128},
+        {QLatin1String("queueBuffer"), 0},
+        {QLatin1String("devicePort"), 4464},
+        {QLatin1String("reverb"), 0},
+        {QLatin1String("limiter"), false},
+        {QLatin1String("compressor"), false},
+        {QLatin1String("quality"), 2},
+        {QLatin1String("captureBoost"), false},
+        {QLatin1String("captureMute"), false},
+        {QLatin1String("captureVolume"), 100},
+        {QLatin1String("playbackBoost"), false},
```

Comment:
Can we remove `captureBoost` and `playbackBoost`? They are valid options, but we've recently started ignoring/retiring them